### PR TITLE
Add internationalization tools.

### PR DESCRIPTION
**Pull request by sergiitk**
#### What's this PR do?
- Install l10n_update module
- Preset language update mode to `"Edited translations are kept, only previously imported ones are overwritten and new translations are added"`
- Add drush "language" plugin
  https://www.drupal.org/project/drush_language
#### How should this be manually tested?

On VM host:
- `vagrant provision`

On VM:
- `drush language-add fr`
- `drush l10n-update-refresh`
- `drush l10n-update`
- `drush language-default fr`
- `drush cc all`
#### Any background context you want to provide?

All Congo site chrome must be translated to French.
#### What are the relevant tickets?

Closes [#DOS-13](https://jira.dosomething.org/browse/DOS-13)

### CODE REVIEW

File: lib/profiles/dosomething/dosomething.info [hunk 1]
```diff
@@ -38,9 +39,8 @@ dependencies[] = field_group
 dependencies[] = file_entity
 dependencies[] = flag
 dependencies[] = i18n
-dependencies[] = libraries
```
Comment (aaronschachter) [inline lib/profiles/dosomething/dosomething.info:41]: Is this intentional?  Off the top head I can't think of anything we're using this module for...

Comment (sergiitk) [inline lib/profiles/dosomething/dosomething.info:41]: Already pushed a quickfix for that.

**Comment by aaronschachter** [comment]
:+1:

**Comment by sergiitk** [comment]
BTW, here's a screenshot
screen shot 2014-07-18 at 7 44 51 pm

**Comment by sergiitk** [comment]
QA is translated to French:
- Login [QA Server](https://sites.google.com/a/dosomething.org/tech/international/deployment-architecture#TOC-QA-Server)
- `cd /var/www/international.dosomething.org/current/html`
- `drush @congo.qa language-add fr`
- `drush @congo.qa l10n-update-refresh`
- `drush @congo.qa l10n-update`
- `drush @congo.qa language-default fr`
- `drush @congo.qa cc all`

**Comment by sergiitk** [comment]
Few images for Wiki article:

image

image

image

image

image

**Pull request merged**